docs/components/control/redirect-to-signin.mdx

```diff
@@ -11,34 +11,66 @@ The `<RedirectToSignIn />` component will navigate to the sign in URL which has
 
 <InjectKeys>
 
-<Tabs type="framework" items={["Next.js", "React", "Remix"]}>
-  <Tab>
-  ```tsx filename="pages/_app.tsx"
-  import {
-    ClerkProvider,
-    SignedIn,
-    SignedOut,
-    RedirectToSignIn,
-  } from "@clerk/nextjs";
-  import { AppProps } from "next/app";
+<Tabs items={["Next.js", "React", "Remix"]}>
+  <CodeBlockTabs options={["App Router", "Pages Router"]}>
+    <Tab>
+    ```tsx filename="app/layout.tsx"
+    import {
+      ClerkProvider,
+      SignedIn,
+      SignedOut,
+      RedirectToSignIn,
+    } from "@clerk/nextjs";
 
-  function MyApp({ Component, pageProps } : AppProps) {
-    return (
-      <ClerkProvider {...pageProps}>
-        <SignedIn>
-          <Component {...pageProps} />
-        </SignedIn>
-        <SignedOut>
-          <RedirectToSignIn />
-        </SignedOut>
-      </ClerkProvider>
-    );
-  }
+    export default function RootLayout({
+      children,
+    }: {
+      children: React.ReactNode;
+    }) {
+      return (
+        <html>
+          <body>
+            <ClerkProvider>
+              <SignedIn>
+                {children}
+              </SignedIn>
+              <SignedOut>
+                <RedirectToSignIn />
+              </SignedOut>
+            </ClerkProvider>
+          </body>
+        </html>
+      );
+    }
+    ```
+    </Tab>
+    <Tab>
+    ```tsx filename="pages/_app.tsx"
+    import {
+      ClerkProvider,
+      SignedIn,
+      SignedOut,
+      RedirectToSignIn,
+    } from "@clerk/nextjs";
+    import { AppProps } from "next/app";
 
-  export default MyApp;
-  ```
-  </Tab>
+    function MyApp({ Component, pageProps } : AppProps) {
+      return (
+        <ClerkProvider {...pageProps}>
```

Comment:
I don't believe pageProps goes on clerkProvider. Only the `<Component>`

Can you check the other page and make this change as well

```suggestion
        <ClerkProvider>
```